client/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+import { Box, Typography, Link } from '@mui/material';
+
+import { SocialLinks } from './SocialLinks';
+import { FooterNav } from './FooterNav';
+
+export const Footer: FC = () => (
+  <Box sx={{
```

Comment:
ur using a box
it has these stuff alreadye exposed as base props
like
display="flex"
again choose a pattern and stick to it
either sx or box properties